models/course_list_view.py

```diff
@@ -0,0 +1,9 @@
+from sqlalchemy import (MetaData, Column, Table, Integer)
+
+
+metadata_obj = MetaData()
+
+
+course_list_view = Table('course_list_view', metadata_obj,
```

Comment:
it' not good idea to create view in this situation. U can do it simply, just select needed data from table 'course'